tests/textcase/lsp/lsp_spec.lua

```diff
@@ -44,5 +44,18 @@ describe("LSP", function()
       local expected_code = test_helpers.read_file("./tests/textcase/lsp/fixtures/snake-case.ts")
       assert.are.same(table.concat(content, "\n"), expected_code)
     end)
+
+    it("Should be triggered on keybinding for constant case", function()
+      test_helpers.execute_keys("/variable_to_be_tested<CR>gaN")
```

Comment:
Should the document be reloaded to keep the tests isolated?

Reply:
Very important point! Thanks for pointing it out :)